xml/quick_configuration.xml

```diff
@@ -156,22 +165,38 @@
         <term>Certificate</term>
          <listitem>
           <para>
+           The <guimenu>Certificate</guimenu> field must be a 
+           Base64-encoded PEM key.
+          </para>
+          <note>
+           <title>Remove the enclosing lines</title>
+           <para>
+            You must enter the certificate body only. This means
+            only the content between the <literal>-----BEGIN CERTIFICATE-----</literal> 
+            and <literal>-----END CERTIFICATE-----</literal> lines
+            but not those lines themselves.
           </para>
+         </note>
          </listitem>
         </varlistentry>
         
         <varlistentry>
          <term>Anonymous</term>
           <listitem>
            <para>
+            Selects if you wish to perform an anonymous bind with
```

Comment:
@danielorf Did I describe this correctly? 
Can you leave DN empty? 
Is it displayed if Anonymous is True?